crates/stacks/src/actor.rs

```diff
@@ -0,0 +1,58 @@
+use std::path::PathBuf;
+
+use kameo::{Actor, actor::ActorRef, message::Message};
+
+use crate::docker::{start_stacks, stop_stacks};
+
+pub struct Worker {
+    pub stacks: bool,
+    pub port: u16,
+    pub config_dir: PathBuf,
+}
+
+pub enum Msg {
+    SetEnabled(bool),
+}
```

Comment:
instead of this, I think I prefer this approach:

```
pub struct SetEnabled(bool);

impl Message<SetEnabled> for Worker { ...
```

that way each message gets its own handler. irrelevant for now since there's only one, but you're able to split the code better, and also have different return values for each message type